wpilibj/src/athena/java/edu/wpi/first/wpilibj/CameraServer.java

```diff
@@ -799,38 +799,4 @@ public void removeCamera(String name) {
       m_sources.remove(name);
     }
   }
-
-  /**
-   * Sets the size of the image to use. Use the public kSize constants to set the correct mode, or
-   * set it directly on a camera and call the appropriate startAutomaticCapture method.
-   *
-   * @deprecated Use setResolution on the UsbCamera returned by startAutomaticCapture() instead.
```

Comment:
Has this one really been deprecated for more than a year?  Or did we just deprecate it last year?